### ISSUE TITLE
Module has no exported member 'xx'

### ISSUE THREAD

**Issue by cobraz** [dderevjanik/wsdl-tsclient#7]
There are multiple references that are not found.

### Example files

filename: GetProjectList2.ts
```typescript
import { Ps1 } from "./Ps1";

export interface GetProjectList {
    /** */
    ps1?: Ps1;
}
```

filename: Ps1.ts
```typescript

export interface Ps {
    /** s:boolean */
    myProjects?: string;
    ... (more fields here)
}
```

Let me know if you want a simple reproduction repo :)

**Comment by dderevjanik** [comment]
Well, there could be a lot of things happening... it would be better to provide an example - if you can
Thanks :)

**Comment by cobraz** [comment]
I'm heading off to bed, but I can create a repro repo tomorrow for you. Meanwhile, here's an example WSDL that is used in my code:

https://api.24sevenoffice.com/Logistics/Product/V001/ProductService.asmx?wsdl

**Comment by dderevjanik** [comment]
I tried that wsdl file and everything seems to be Ok.
Can you send me exact file ?

**Comment by cobraz** [comment]
I've created a reproduction here:
https://github.com/cobraz/wsdl-tsclient-repro-duplicate-models

**Comment by dderevjanik** [comment]
Thanks ! 
I read that code and I think that problem is casued because of `definitionsList`, which holds all reference to definitions (to be able to avoid cyclic references), is defined as global and can persists between sessions. Much worse if you're not using `await generateClient`. 

This `global` will be fixed in newer version of parser/generated (currently in `dev` branch)

**Comment by dderevjanik** [comment]
Try to use `await generateClient`

**Comment by dderevjanik** [comment]
Well, I cloned reproduction repository, edited `generate.js`:

```js
const wsdl = require('wsdl-tsclient');

(async function () {

    await wsdl.generateClient('wsdl/AccountService.wsdl', 'src/generated/AccountService');
    await wsdl.generateClient('wsdl/AttachmentService.wsdl', 'src/generated/AttachmentService');
    await wsdl.generateClient('wsdl/Authenticate.wsdl', 'src/generated/Authenticate');
    await wsdl.generateClient('wsdl/BudgetService.wsdl', 'src/generated/BudgetService');
    await wsdl.generateClient('wsdl/ClientService.wsdl', 'src/generated/ClientService');
    await wsdl.generateClient('wsdl/CompanyService.wsdl', 'src/generated/CompanyService');
    await wsdl.generateClient('wsdl/FileInfoService.wsdl', 'src/generated/FileInfoService');
    await wsdl.generateClient('wsdl/FileService.wsdl', 'src/generated/FileService');
    await wsdl.generateClient('wsdl/InvitationService.wsdl', 'src/generated/InvitationService');
    await wsdl.generateClient('wsdl/InvoiceService.wsdl', 'src/generated/InvoiceService');
    await wsdl.generateClient('wsdl/PaymentService.wsdl', 'src/generated/PaymentService');
    await wsdl.generateClient('wsdl/PersonService.wsdl', 'src/generated/PersonService');
    await wsdl.generateClient('wsdl/ProductService.wsdl', 'src/generated/ProductService');
    await wsdl.generateClient('wsdl/ProjectService.wsdl', 'src/generated/ProjectService');
    await wsdl.generateClient('wsdl/SalesOppService.wsdl', 'src/generated/SalesOppService');
    await wsdl.generateClient('wsdl/TemplateService.wsdl', 'src/generated/TemplateService');
    await wsdl.generateClient('wsdl/TimeService.wsdl', 'src/generated/TimeService');
    await wsdl.generateClient('wsdl/TransactionService.wsdl', 'src/generated/TransactionService');

})();
```

And then run `rm -rf ./src && node generate.js`, then there we no more duplicates

**Comment by dderevjanik** [comment]
@cobraz error with generating duplicated definitions should be fixed in [Release 1.1.3](https://github.com/dderevjanik/wsdl-tsclient/releases/tag/1.1.3)

**Comment by cobraz** [comment]
Thanks! 🙏

Let's close this then? :)
